layouts/partials/brand.html

```diff
@@ -1,6 +1,6 @@
 <header class="brand">
-  <h1>{{ .Site.Title}}</h1>
+  <a href="/"><h1>{{ .Site.Title}}</h1></a>
```

Comment:
remplace `href="/"` by `href="{{ .Site.BaseURL }}"`

Reply:
Done!

Reply:
You don't have to merge the link title if you don't want. I was only doing it for my site, but it ended up in the same pull request.